### PR TITLE
Radar Chart added

### PR DESCRIPTION
**Pull request by joshi-kaushal**
## Related Issue
  - Added a project that creates a Radar Chart based on the user inputs

### Checklist:
- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] New and existing unit tests pass locally with my changes.

### Screenshots
image

### CODE REVIEW

**Review by Rizwan-S — CHANGES_REQUESTED** [review]
Please remove 'linechart.html' file

**Comment by joshi-kaushal** [comment]
@Rizwan-S along with its image and mention in script.js or just html file?

**Comment by joshi-kaushal** [comment]
@Rizwan-S please have a look.

**Review by Rizwan-S — CHANGES_REQUESTED** [review]
Please remove the 'password.html', 'password-generator.jpg' files

File: script.js [hunk 1]
```diff
@@ -2041,16 +2041,14 @@ let projectData = [
     projectUrl: "public/Double-Vertical-Slider.html",
   },
   {
-
-    projectName: "Bar Chart",
-    projectImage: "assets/images/barchart.png",
-    projectUrl: "public/barchart.html",
```
Comment (Rizwan-S) [inline script.js:2047]: Bar Chart project is already present. Please don't remove this code.

**Comment by joshi-kaushal** [comment]
@Rizwan-S please check, i did `git pull` once so has some updated info.

**Review by Rizwan-S — CHANGES_REQUESTED** [review]
Please move the radar.html file to public folder.

**Review by Rizwan-S — APPROVED** [review]
Excellent!!!

**Pull request merged**